x-pack/plugins/enterprise_search/public/applications/shared/kibana/kibana_logic.ts

```diff
@@ -63,6 +64,7 @@ export interface KibanaLogicProps {
   isSidebarEnabled: boolean;
   lens?: LensPublicStart;
   ml?: MlPluginStart;
+  licensing?: LicensingPluginStart;
```

Comment:
Why are we adding the licensing plugin to Kibana Logic? That doesn't seem necessary.

Reply:
Updated